### PR TITLE
Hide Individual Quests

### LINKED ISSUE #330: [Feature Request] Quests hide
Would be great to have a [Hide] ability for Quests. This way we can go for Quests hunting and easily mark discovered.

### PR DESCRIPTION
**Pull request by versx**
Fixes: #330 
image

### CODE REVIEW

**Review by TurtIeSocks — CHANGES_REQUESTED** [review]

File: static/js/index.js [hunk 1]
```diff
@@ -3471,7 +3487,16 @@ function getPokestopPopupContent (pokestop) {
 
     const questReward = pokestop.quest_rewards ? pokestop.quest_rewards[0] : {};
     if (pokestop.quest_type !== null) {
-        content += `<a title="Filter Quest" href="#" onclick='addQuestFilter(${JSON.stringify(questReward.info)}, false);return false;'><div class="exclude">[Exclude]</div></a>`;
+        content += `
+        <div class="pokemon-timer-hide-exclude">
+        <table class="table-fourth-row">
+          <tr>
+            <td><a title="Filter Quest" href="#" onclick="addQuestFilter(${JSON.stringify(questReward.info)}, false);return false;"><b>[Exclude]</b></a></td>
```
[the remark this answers is not in the document]
Comment (versx) [reply static/js/index.js:3494]: Interesting, I didn't experience that but will commit the suggestion non-the-less

**Review by TurtIeSocks — APPROVED** [review]

File: static/js/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mygod) [reply static/js/index.js]: Because JSON uses double quotes.

**Comment by Mygod** [comment]
I have a separate comment. When there is both quests and invasions on the stop, it is not clear if you are excluding/hiding quests or invasions.

**Comment by versx** [comment]
> I have a separate comment. When there is both quests and invasions on the stop, it is not clear if you are excluding/hiding quests or invasions.

That's a good point, I'll update the text

**Pull request merged**